cvat/apps/engine/serializers.py

```diff
@@ -496,3 +498,75 @@ class LogEventSerializer(serializers.Serializer):
 
 class AnnotationFileSerializer(serializers.Serializer):
     annotation_file = serializers.FileField()
+
+class ReviewSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.Review
+        fields = '__all__'
+        read_only_fields = ('id', )
+        write_once_fields = ('job', 'reviewer', 'assignee', 'estimated_quality', 'status', )
+        ordering = ['-id']
+
+class ReviewSummarySerializer(serializers.Serializer):
+    reviews = serializers.IntegerField(read_only=True)
+    average_estimated_quality = serializers.FloatField(read_only=True)
+    issues_unsolved = serializers.IntegerField(read_only=True)
```

Comment:
What is the reason to do the logic on server?

Reply:
On a task view we need a short summary only. I am not sure we should send all the issues for all the job there to collect this statistics on client. 
![Screenshot from 2020-11-02 12-12-29](https://user-images.githubusercontent.com/40690378/97850199-bc3b2700-1d04-11eb-86ff-ce78a92b6f35.png)

Reply:
And there might be issues related with permissions (job assignees mustn't see issues from other jobs, but at the same time they can see short summary of other jobs).

Reply:
Let's don't optimize too early. I don't think that we need it till we have clear understanding of that. It will complicate REST API . Probably the best place for the logic at the moment is cvat-core.

Reply:
Redesigned